plugins/services/src/js/containers/services/ServiceTreeView.js

```diff
@@ -110,32 +102,29 @@ class ServiceTreeView extends React.Component {
   }
 
   getTabs() {
-    const { quota } = ConfigStore.get("config").uiConfiguration.features || {};
     // Workaround for Cypress.
     const createModalOpen = this.context.router.routes.some(
       ({ isFullscreenModal }) => isFullscreenModal
     );
 
-    if (!quota || createModalOpen) {
+    if (createModalOpen) {
       return [];
     }
     const { serviceTree } = this.props;
-    if (
-      serviceTree.isRoot() ||
-      typeof serviceTree.getEnforceRole() === "boolean"
-    ) {
+    const id = serviceTree.getId();
+    if (id.split("/").length <= 2) {
```

Comment:
This is just an easy way to check if the group id is `/` or `/<name>` ie root or top-level. 

I have some additional work in progress that will remove this check in favor of a `hasQuota` check for top-level groups.

Reply:
We have a `isRoot` function for `ServiceTree`.